bc_obps/.env.example

```diff
@@ -0,0 +1,6 @@
+SECRET_KEY=change-this-secret-key
+DB_NAME=your_db_name
```

Comment:
Might be helpful to add a bullet in the readme to tell people to create a new db, and what the name should be if it matters